### ISSUE TITLE
request: volume bounce intensity settings at bake

### ISSUE THREAD

**Issue by laurentopia** [guycalledfrank/bakery-issues#85]
volumes are great for high fps on mobile
but if i want to change the effect they have on objects, for example boost intensity, which is favorable for cartoony look, i have to change the bounce intensity of the light which will then affect the environment.
adding some control over volume baking would help that.

**Comment by guycalledfrank** [comment]
Boosting the intensity of the whole volume is easier to do directly in the shader IMHO. Just multiply it there?

**Comment by laurentopia** [comment]
yes for the overall brightness but not for the indirect component, which is the one i want to boost to give a certain cartoony look
i use volume for lighting, volumes include direct and indirect lighting mixed

**Comment by guycalledfrank** [comment]
So you want to increase indirect intensity and only for volumes, but not the lightmaps? It'll give pretty inconsistent results. Maybe OK on some scenes, not OK on other scenes.

**Comment by laurentopia** [comment]
Correct,
 it allows characters and objects to stand out like in a cartoon
  and give a glowy look, a bit like free SSS
   it'll look a bit like this
    ![image](https://user-images.githubusercontent.com/26075468/115306231-60325680-a11c-11eb-9b5a-848cfd282916.png)
